engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/SetRemovalTimeToHistoricProcessInstancesDto.ftl

```diff
@@ -16,9 +16,25 @@
   <@lib.property
       name = "hierarchical"
       type = "boolean"
-      last = true
       desc = "Sets the removal time to all historic process instances in the hierarchy.
               Value may only be `true`, as `false` is the default behavior."/>
 
+  <@lib.property
+      name = "updateInChunks"
+      type = "boolean"
+      desc = "Processes removal time updates in chunks, taking into account defined limits.
+              This can lead to multiple executions of the resulting jobs and prevents the database
+              transaction from timing out by limiting the number of rows to update.
+              Value may only be `true`, as `false` is the default behavior."/>
```

Comment:
❓Can we add here if `updateChunkSize` is not passed, the default from the process engine config is used? In the `updateChunkSize` documentation, we could also mention that this falls back to the engine config if undefined.

Reply:
Absolutely, yeah 👍

Reply:
Let me know what you think about the separate suggestions I made below 🙂

Reply:
@tasso94, I committed the proposals to make progress. Happy for any feedback you have on the new descriptions 👍

Reply:
I like it 👍

Reply:
🚀